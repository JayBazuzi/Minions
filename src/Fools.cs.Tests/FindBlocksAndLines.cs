using ApprovalTests.Reporters;
using Fools.cs.AST;
using NUnit.Framework;

namespace Fools.cs.Tests
{
	[TestFixture, UseReporter(typeof(DiffReporter))]
	public class FindBlocksAndLines
	{
		[Test]
		public void should_find_nothing_in_an_empty_file()
		{
			string.Empty.find_blocks().should_parse_correctly();
		}

		[Test]
		public void file_with_a_single_empty_block_should_parse()
		{
			@"
some block:
	pass
".find_blocks().should_parse_correctly();
		}
	}
}
