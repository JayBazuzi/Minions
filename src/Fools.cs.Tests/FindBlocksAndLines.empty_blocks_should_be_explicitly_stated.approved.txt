/** Declarations **/

[
  {
    "name": "funky_function",
    "body": []
  }
]
